.circleci/config.yml

```diff
@@ -7,31 +7,39 @@ jobs:
             - checkout
             - restore_cache:
                   keys:
-                      - v2-dependencies-{{ checksum "package.json" }}-{{ checksum "rn-vector/package.json"}}
-                      - v2-dependencies-
+                      - v2-dependencies-{{ checksum "package.json" }}-{{ checksum "mui/package.json" }}-{{ checksum "rn-vector/package.json"}}
             - run:
                   name: Install Dependencies
                   command: |
                       yarn install
-                      cd rn-vector && yarn install
+                      cd rn-vector && yarn install && cd ..
+                      cd mui && yarn install
             - save_cache:
                   paths:
                       - node_modules
-                  key: v2-dependencies-{{ checksum "package.json" }}-{{ checksum "rn-vector/package.json"}}
+                  key: v2-dependencies-{{ checksum "package.json" }}-{{ checksum "mui/package.json" }}-{{ checksum "rn-vector/package.json"}}
             - run:
-                  name: Build Packages
-                  command: | 
-                      yarn build
-                      cd rn-vector && yarn build
+                  name: Build Font and SVG Icons
+                  command: yarn build
+            - run:
+                  name: Build RN Vector Icons
+                  command: cd rn-vector && yarn build
+            - run:
+                  name: Build MUI
+                  command: cd mui && yarn build
             - run:
                   name: Verify SVG Build Artifacts
                   command: yarn test
+            - run:
+                  name: Verify MUI Artifacts
+                  command: cd mui && yarn test
             - persist_to_workspace:
                   root: .
                   paths:
                       - iconfont
                       - svg
                       - rn-vector
+                      - mui/dist
 
     publish:
```

Comment:
let's add the automatic tagging while we are in here

Reply:
Just added this now.